site/styles/_awards.scss

```diff
@@ -6,13 +6,14 @@
     gap: 30px 104px;
     grid-auto-rows: 1fr;
     margin-bottom: 51px;
+    margin-top: -14px;
```

Comment:
Is there a reason why you can't remove/reduce the margin-bottom on the elements above the awards instead of using this forbidden magic?

Reply:
Its to satisfy this scenario 
![image](https://user-images.githubusercontent.com/32156362/187809355-9b3afb94-38be-4615-961a-6cebde2767d9.png)

Reply:
Nope, this problem would remain if see more/less was just a static block element at the bottom of the element switcher

Reply:
Is it supposed to look like this?
![image](https://user-images.githubusercontent.com/10665859/187824409-eb669626-515d-4c1b-8883-1a385f37a35c.png)

That gap is not the same as your screenshot.  Am I confused?

Reply:
It is. Oh the screenshot is of the google doc, which is how it looked before. The red squares are what the gaps should be, not measurements.

Reply:
If that's how it's supposed to look then resolved!